src/view/com/composer/text-input/web/EmojiPicker.web.tsx

```diff
@@ -6,15 +6,17 @@ import {
   View,
 } from 'react-native'
 import Picker from '@emoji-mart/react'
+import EventEmitter from 'eventemitter3'
 
 import {atoms as a} from '#/alf'
-import {textInputWebEmitter} from '../TextInput.web'
```

Comment:
I have to move this out of TextInput because it is now used everywhere that EmojiPicker is used.